src/orders/types.ts

```diff
@@ -58,11 +59,42 @@ export const OrderEventsEnum = {
   ...OrderEventsEnumWithoutTransactions,
 };
 
+/** Type guard for order with transactions */
 export const isOrderWithTransactions = (
   _order: any,
   featureFlag: boolean,
 ): _order is OrderDetailsWithTransactionsFragment => featureFlag;
 
+/** Check if order has transactions & feature flag enabled */
+export const orderHasTransactions = (
+  order: any,
```

Comment:
**question**: Is there a reason that the order is `any` type?

Reply:
Actually, you're right this could be an `unknown` instead